projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -107,8 +107,8 @@ export class GaugeComponent implements OnChanges {
     return this.buildArcGenerator()({
       innerRadius: radius - GaugeComponent.GAUGE_RING_WIDTH,
       outerRadius: radius,
-      startAngle: -Math.PI / 2,
-      endAngle: -Math.PI / 2 + (inputData.value / inputData.maxValue) * Math.PI
+      startAngle: -Math.PI / 2 - GaugeComponent.EXTRA_ARC_ANGLE,
+      endAngle: -Math.PI / 2 - GaugeComponent.EXTRA_ARC_ANGLE + (inputData.value / inputData.maxValue) * Math.PI
```

Comment:
This seems incorrect to me. 

case 1:  `inputData.value = 0`,  the end angle would be `-Math.PI / 2 - GaugeComponent.EXTRA_ARC_ANGLE` which is same as start angle. So the arc will be completely empty. Expected outcome.

case 2: `inputData.value = inputData.maxValue`, in this case the end angle of value arc should be equal to max end angle of background arc, that is `Math.PI / 2 + GaugeComponent.EXTRA_ARC_ANGLE`. Line 111, would make it `-Math.PI / 2 - GaugeComponent.EXTRA_ARC_ANGLE + 1 * Math.PI = Math.PI / 2 - GaugeComponent.EXTRA_ARC_ANGLE`

So the value arc would never be completely full.

Reply:
Good catch, I will fix that